samples/cancel-batch-prediction-job.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+async function main(batchPredictionJobId, project, location = 'us-central1') {
+  // [START aiplatform_cancel_batch_prediction_job]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.\
+   * (Not necessary if passing values as arguments)
```

Comment:
We generally expect folks to be using these from the embedded snippets on Cloud site, and almost never cloning the repo and running these locally. For those cases, we're kind of ok letting users figure out they don't need to fill out those fields.  I don't think we need to add the `\ (No necessary ...` bit here, as we don't add that anywhere else.

Reply:
Sounds good.